app/src/main/java/com/mapbox/maps/testapp/examples/SecondaryDisplaySimpleMapActivity.kt

```diff
@@ -0,0 +1,39 @@
+package com.mapbox.maps.testapp.examples
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.mapbox.maps.Style
+import com.mapbox.maps.testapp.R
+import kotlinx.android.synthetic.main.activity_simple_map.*
+
+/**
+ * Example of displaying a map.
+ */
+class SecondaryDisplaySimpleMapActivity : AppCompatActivity() {
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.activity_simple_map)
+    mapView.getMapboxMap().loadStyleUri(Style.MAPBOX_STREETS)
```

Comment:
Just as an idea: guess we could also add button to perform camera fly on secondary screen.